pkg/filter/filter.go

```diff
@@ -13,12 +13,14 @@ const (
 	invulnerable string = "invulnerable"
 	noResults    string = "no-results"
 	unknown      string = "unknown"
+	localCheck   string = "local-check"
```

Comment:
Question: What's this for? I can't actually find where it's used in this file

Reply:
Ah, yes, you're right. This is an artifact from when I had the bug in the code. Will remove. Thanks!